reverseforwardrecord/.github/workflows/get_dns_records.yaml

```
@@ -0,0 +1,36 @@
+name: Cloud DNS tests
+
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - "reverseforwardrecord/**"
+      - ".github/workflows/get_dns_records.yaml"
+
+jobs:
+  test_and_lint:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ["3.x", "3.10", "3.11"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: "pip"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r reverseforwardrecord/requirements.txt
+
+      - name: Run tests
+        run: cd reverseforwardrecord/ && python3 -m pytest .
+
+      - name: Analyse with pylint
+        run: |
+          cd reverseforwardrecord && pylint $(git ls-files *.py)
```

```suggestion
          cd reverseforwardrecord && pylint .
```
This should function the same and is less complicated. This is only a suggestion.